static/usage/action-sheet/basic/vue.md

```diff
@@ -0,0 +1,70 @@
+```html
+<style>
+  .container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    flex-direction: column;
+  }
+
+  code {
+    white-space: pre-wrap;
+  }
+</style>
+
+<template>
+  <div class="container">
+    <ion-button @click="presentActionSheet">Open</ion-button>
+    <code>{{result}}</code>
+  </div>
+</template>
+
+<script lang="ts">
+  import { IonButton, actionSheetController } from '@ionic/vue';
+
+  export default {
+    components: { IonButton },
+    data() {
+      return {
+        result: '',
+      };
+    },
+    methods: {
```

Comment:
Any particular reason why the Options API was chosen over the Composition API?

Reply:
I thought we were waiting until all component playgrounds exist, before moving to the Composition API usage?

Reply:
I think that was for using `script setup`, not the Composition API. Our current usage looks to be inconsistent:

Composition API: https://ionicframework.com/docs/api/accordion#toggle-accordions
Options API: https://ionicframework.com/docs/api/alert#controller

Reply:
Ahh good distinction - I'll get that fixed here + get `ion-alert` updated as well.